deployment/terraform/modules/haystack-datastores/kubernetes/main.tf

```diff
@@ -16,7 +16,7 @@ module "es" {
   kubectl_executable_name = "${var.kubectl_executable_name}"
   kubectl_context_name = "${var.kubectl_context_name}"
   node_selecter_label = "${var.node_selecter_label}"
-  memory_limit = "${var.memory_limit}"
+  memory_limit = "400"
```

Comment:
lets keep using var.memory_limit, this variable lets us override the memory allocated to pods in case the vm has less memory